content/kubermatic/master/startio/cheat_sheets/_index.en.md

```diff
@@ -0,0 +1,4 @@
++++
+title = "Cheat Sheets"
+weight = 60
++++
```

Comment:
Tiny nit: we should add a short introductory text/description for this section. We already have that for other sections.